src/tests/runTest.ts

```diff
@@ -18,13 +18,41 @@
 // All tests ran here are ones that require the vscode API
 import * as path from 'path'
 import * as cp from 'child_process'
+import { parseArgs } from 'util'
 import {
   runTests,
   resolveCliArgsFromVSCodeExecutablePath,
   downloadAndUnzipVSCode,
 } from '@vscode/test-electron'
 
+// VS Code version 1.60.2, released October 4, 2021, is the earliest version
+// supported as indicated in package.json engines/vscode, so that version is a
+// reasonable default.
+const DEFAULT_DAFFODIL_TEST_VSCODE_VERSION = '1.60.2'
```

Comment:
Can we pull this out of package.json or yarn.lock so we don't need to remember to update this file when one of those files is updated?

Reply:
Good idea!  I'll get that added.